users/test_login_server.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from django.contrib.auth.models import User
+
+
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+
+
+@pytest.fixture
+def new_user():
+    user = User(username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, password=PASSWORD, email=EMAIL)
+    user.set_password(PASSWORD)
+    return user
+
+
+@pytest.mark.django_db()
+class TestProfileServerResponse:
+    # Test that the login webpage loaded
+    def test_login_entry_point(self, client):
+        response = client.get("/login/")
```

Comment:
According to your checklist, you should use `'` or `"` but you should choose one \
There is a use of both here, so please align and use one syntax for all quotation marks in the file

Reply:
Done 😄